### ISSUE TITLE
Assistivetech

### ISSUE THREAD

**Issue by Mukkund10** [Mukkund10/AccessibilityToolTech#1]
While our repository hosts technologies of accessible tech related to social impact research, we've identified an opportunity to improve accessibility. Currently, some papers may lack proper formatting, making it challenging for users with visual impairments to access the content easily. We aim to address this issue and make our accessible tool more inclusive.
